marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/hub-entities.xqy

```diff
@@ -177,12 +218,15 @@ declare %private function hent:fix-options($nodes as node()*)
       default return $n
 };
 
-declare function hent:dump-search-options($entities as json:array)
+declare function hent:dump-search-options($entities as json:array, $params as map:map)
```

Comment:
This looks good, but one small tweak - unless we have a large number of params to pass around, it's better to explicitly declare the parameter here - i.e. "$for-explorer as xs:boolean?" (I would use "?" so it's fine to pass in an empty sequence).

Reply:
Changed the method signature to boolean in the new commits